readthedocs/projects/tasks.py

```diff
@@ -229,6 +222,43 @@ def setup_vcs(self):
             raise BuildEnvironmentError('Failed to import project',
                                         status_code=404)
 
+    def get_bash_env(self):
+        """
+        Get bash environment variables used for all builder commands.
```

Comment:
We need some way to name them different than the 2 other types of environments we use. Any suggestions?